docs/migration/5_0-generated-typescript-changes-doc.md

```diff
@@ -22065,7 +22362,7 @@ dialogClose: Observable<string>
 Current version: 
 
 ```
-dialogClose: Observable<string | undefined>
+dialogClose: Observable<any | undefined>
```

Comment:
is it breaking change?

Reply:
It can now be undefined while it could not before.  To be on the safe side I'd leave it in the migrations.

Reply:
the type is widen than before. So, I think it is not breaking change. But, nothing is hurt if we leave it in migrations.